.docs/env-vars.md

```diff
@@ -0,0 +1,23 @@
+# How to define and consume environment variables
+
+## How to define
+
+1. Add the variables with their values to `.env.local` and DO NOT PUSH this file.
```

Comment:
klopt het dat het naar .env.local moet en niet naar .env zonder iets erachter? want dat is dan wel anders dan voorheen, dus moeten we dan wel allemaal doorvoeren

Reply:
Nee, `.env` zal gewoon werken, dat zal ik erbij zetten. Ik vind `.env.local` een betere keuze: is semantisch logisch en hij zal lokaal eventuele andere verdwaalde .env files overriden. Maar dat is niet zwaarwegend genoeg om het af te gaan dwingen